secret.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-secret
+type: Opaque
+data:
+  MYSQL_ROOT_PASSWORD: cm9vdHJvb3Q= #rootroot
+  MYSQL_USER: dGVzdHVzZXI= #testuser
+  MYSQL_PASSWORD: cGFzc3dvcmQ= #password
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)